### DISCUSSION TITLE
How to set SnackBar duration. [Question]

### DISCUSSION THREAD

**Discussion by deco3net** [flet-dev/flet#1527] [Q&A]
### Question

Officially, the change in the display time of the SnackBar is described as follows.

https://flet.dev/docs/controls/snackbar
[The number of milliseconds that the SnackBar stays open for. Defaults to 4000 (4 seconds) when not set.]

I want to change from the default of 4000 (4 seconds) to 15000 (15 seconds).

However, it does not specify exactly how to do this.
I have tried various methods, but the stated time remains unchanged.

What code can I add to change it?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Just like all the other props: `SnackBar(duration=15000, ...)`

**Reply by deco3net** [reply]
Thanks for the reply.

page.snack_bar = ft.SnackBar(ft.Text("Text of some kind."))
page.snack_bar.open = True
page.update()

With this code of mine the snack bar is displayed, but I cannot change the display time.
Where do I need to add the [duration] code?

**Reply by deco3net** [reply]
Hi.
Updating the flet to the latest version has reflected the [duration] setting.
Thanks.

**Reply by deco3net** [reply]
page.snack_bar = ft.SnackBar(
                content=ft.Text("Hello, world!"),
                action="Alright!",
                duration = 15000,
                open=True,
            )
            page.update()
